build.gradle.kts

```diff
@@ -11,19 +11,21 @@ import org.jlleitschuh.gradle.ktlint.KtlintExtension
 import java.net.URL
 
 buildscript {
+    apply(from = "buildSrc/extra.gradle.kts")
```

Comment:
I don't think we get much from extracting the `androidGradlePlugin` and `kotlinPlugin` into a separate file. Also `extra.gradle.kts` will cause us to set the `repositories` block multiple times. Can you remove this part?

Reply:
Removed.
But for using `Gradle` and `Kotlin` api in `buildSrc`, should add theirs dependencies and repos in buildSrc's `build.gradle.kts`, therefore repos have been declared many times. This is the reason why I add a `extra.gradle.kts` and import with `apply(from = ...)`

Reply:
@Goooler Ah sorry I missed that. I'll factor out the versions into gradle variables in a separate PR.